src/DIRAC/Resources/Computing/BatchSystems/SLURM.py

```diff
@@ -191,7 +191,7 @@ def getJobStatus(self, **kwargs):
         lines = output.strip().split("\n")
         jids = set()
         for line in lines:
-            jid, status = line.split()
+            jid, status = line.split(',')
```

Comment:
```suggestion
            jid, status = line.split(",")
```
to make `black` happy